plugins/c8y_configuration_plugin/src/child_device.rs

```diff
@@ -9,11 +9,15 @@ use tracing::error;
 use crate::config::FileEntry;
 use crate::error::ChildDeviceConfigManagementError;
 
-// FIXME move this to tedge config
 #[cfg(test)]
-pub const FILE_TRANSFER_ROOT_PATH: &str = "/tmp";
+use tedge_config::FILE_TRANSFER_ROOT_PATH_TEST;
+#[cfg(test)]
+pub const FILE_TRANSFER_ROOT_PATH: &str = FILE_TRANSFER_ROOT_PATH_TEST;
```

Comment:
```suggestion
#[cfg(test)]
pub const FILE_TRANSFER_ROOT_PATH: &str = "/tmp;
```
As per my earlier comment.